i8_terminal/commands/screen/screen_search.py

```diff
@@ -33,7 +33,7 @@
 def prepare_screen_df(
     conditions: List[str], metrics: str, sort_by: Optional[str], sort_order: Optional[str]
 ) -> Tuple[List[str], Optional[pd.DataFrame]]:
-    max_count = 20
+    max_count = 520
```

Comment:
Have you test it with a few screening scenarios? Does it make sense to have such a high count for all scenarios?